### PR TITLE
feat: add claim kint guide

### PR DESCRIPTION
**Pull request by nud3l**

### CODE REVIEW

File: docs/kintsugi/crowdloans.md [hunk 1]
```diff
@@ -47,3 +46,36 @@ We recommend the following ways to participate:
...
+* Contributions via https://polkadot.js.org/apps/#/, please use the account that you have used for the contribution
+* Contributions via the Bifrost SALP (https://bifrost.app/vcrowdloan?tab=ksm), please use the account that you have used for the contribution on the Bifrost app
+
+Other contributions, e.g., via exchanges will airdrop KINT tokens through other means. If you have contributed via any other means than the ones listed above, please contact the provider of the contribution for the KINT airdrop.
+
+### 1. Go to the KINT claim website at [ https://kintsugi.interlay.io/](https://kintsugi.interlay.io/)
+
+Scroll down to the claim form. If you have contributed to the Kintsugi crowdloan, you will see your estimated KINT airdrop together with your KSM locked.
+
+Click the checkbox accept the terms and conditions.
```
Comment (alexeiZamyatin) [inline docs/kintsugi/crowdloans.md]:
```suggestion
You MUST read and accept the terms and conditions to qualify for the airdrop.
```

File: docs/kintsugi/crowdloans.md [hunk 2]
```diff
@@ -47,3 +46,36 @@ We recommend the following ways to participate:
...
+
+![Claim KINT](../_assets/img/kintsugi/claim_kint_1.png)
+
+### 2. Sign the Terms and Conditions by clicking the "Claim KINT" button
+
+You will be asked to sign the terms and conditions via the polkadot.js extension.
+
+?> Signing the T&Cs is free of charge, i.e., there are no transaction fees for signing the T&Cs.
```
Comment (alexeiZamyatin) [inline docs/kintsugi/crowdloans.md]:
```suggestion
?> There are **no** transaction fees for signing the T&Cs.
```

**Comment by alexeiZamyatin** [comment]
Added 2 comments. 

Missing: a section on how people can check their KINT balance.

**Comment by nud3l** [comment]
> Added 2 comments.
> 
> Missing: a section on how people can check their KINT balance.

I think you did not submit your review. Can you check if you pressed the button since I don't see any comments? Thanks!

**Review by alexeiZamyatin — CHANGES_REQUESTED** [review]

**Review by alexeiZamyatin — APPROVED** [review]

**Pull request merged**